packages/hdwallet-native/src/sui.ts

```
@@ -0,0 +1,61 @@
+import * as core from "@shapeshiftoss/hdwallet-core";
+
+import { Isolation } from "./crypto";
+import { SuiAdapter } from "./crypto/isolation/adapters/sui";
+import { NativeHDWalletBase } from "./native";
+
+export function MixinNativeSuiWalletInfo<TBase extends core.Constructor<core.HDWalletInfo>>(Base: TBase) {
+  return class MixinNativeSuiWalletInfo extends Base implements core.SuiWalletInfo {
```

ESLint

[error] 'MixinNativeSuiWalletInfo' is already declared in the upper scope on line 7 column 17.

(@typescript-eslint/no-shadow)